include/alpaka/core/UniformCudaHip.hpp

```diff
@@ -90,6 +90,9 @@ namespace alpaka
                     {
                         rtCheck(error, ("'" + std::string(cmd) + "' returned error ").c_str(), file, line);
                     }
+                    else
+                        // reset the last error to avoid propagation to the next CUDA/HIP API call
```

Comment:
I would prefer braces here, especially since the comment makes it multiline.

Reply:
I will change it